backend/src/controllers/encounter.controller.ts

```diff
@@ -1,80 +1,84 @@
+import { NextFunction, Request, Response } from 'express';
+import httpStatus from 'http-status';
+import encounterService from '../services/encounter.service';
+import logger from '../utils/logger';
+import { EncounterModel } from '../models/encounter.model';
+import { getUserByAuthId } from '../services/user.service';
 
-     // Pass data to service and attempt to save
-     const createdEncounter = await encounterService.createEncounter(encounterReq);
-     // Notify frontend that the operation was successful
-     res.status(httpStatus.CREATED).json(createdEncounter).end();
-   } catch (e) {
-     next(e);
-   }
- };
-export const updateEncounter = async (
-    req: Request,
-    res: Response,
-    next: NextFunction,
-): Promise<any> => {
-    logger.info("PUT /encounter/:id request from frontend");
-
-    try {
-        // check authorization is not a null object (this is necessary check for lint)
-        if (req.headers.authorization == null)
-            return res.status(httpStatus.NOT_FOUND).end()
-
-        const encounterIdToUpdate = req.params.id
-        const firebaseAuthId = req.headers.authorization['user_id']
-
-        // check that user exists with that firebase auth_id, and the user specified encounter actually exists
-        const userByAuthId = await getUserByAuthId(firebaseAuthId);
-        if(!userByAuthId || !isEncounterExistsInUser(userByAuthId, encounterIdToUpdate))
-            return res.status(httpStatus.NOT_FOUND).end()
-
-        // update encounter
-        const newEncounterData = getEncounterFromReqBody(req.body);
-        const updatedEncounter = await encounterService.updateEncounter(encounterIdToUpdate, newEncounterData);
-        return res.sendStatus(updatedEncounter?httpStatus.NO_CONTENT:httpStatus.NOT_FOUND).end();
-    } catch (e) {
-        next(e);
-    }
-};
-
- 
- // Util function that won't be needed regularly
+// Util function that won't be needed regularly
 const getEncounterFromReqBody = (body: any) => {
-    const encounter: EncounterModel = {
-        date: body.date,
-        location: body.location,
-        description: body.description,
-        persons: body.persons
-    }
+  const encounter: EncounterModel = {
+    date: body.date,
+    location: body.location,
+    description: body.description,
+    persons: body.persons,
+    title: body.title,
+  };
 
-   return encounter;
- }
+  return encounter;
+};
 
 /**
  * searches for encounter in user by encounter id
  * @param user user which encounter should be found from
  * @param encounterId id of the encounter to be searched from the user
  */
 const isEncounterExistsInUser = (user, encounterId: string): boolean => {
-    const findEncounter = user.encounters.filter(e=>e.toString() === encounterId)
-    return findEncounter.length!==0
-}
\ No newline at end of file
+  const findEncounter = user.encounters.filter((e) => e.toString() === encounterId);
+  return findEncounter.length !== 0;
+};
+
+export const createEncounter = async (
```

Comment:
Shouldnt this method be added by the `POST /encounters` issue?

Reply:
I think he might have just ran a lint-fix

Reply:
oh yeah it was added by that issue, hajin is correct, this was just edited by a lint-fix